### ISSUE TITLE
Easy way to create additional viewers from GUI

### ISSUE THREAD

**Issue by sofroniewn** [napari/napari#3955]
## 🚀 Feature

A simple way to create additional viewers from the GUI - for example `cmd+N`.

## Motivation

Multiple viewers is an increasingly common pattern, and we're doing more and more to improve support for it. Having a simple way for GUI users to create additional viewers will ensure that our support becomes 100% and establish the multi-viewer usecase as a first class citizen.

## Alternatives

We prevent the multi-viewer use case from API and GUI.

## Additional context

Better support for multiple viewers could help us show our examples in new viewers.

What do @jni @tlambert03 @brisvag @JoOkuma and others who use multiple viewers think? I know i've always been one of the most negative about the multiviewer usecase, but I think so many people want to use them we should add complete support

**Comment by tlambert03** [comment]
yep, I'm supportive.  even if we someday move to a single MainWindow model (like photoshop or something), everything we do to support multiple viewer models will still be useful.  The lift required to turn a single window into a multi-viewer environment is far greater than just making sure we handle multiple windows well for now... and I think most of that work translates

**Comment by sofroniewn** [comment]
> even if we someday move to a single MainWindow model (like photoshop or something), everything we do to support multiple viewer models will still be useful. The lift required to turn a single window into a multi-viewer environment is far greater than just making sure we handle multiple windows well for now... and I think most of that work translates

Totally agree - I made #3956 to discuss some of the tabbing considerations separately.
